src/databricks/labs/ucx/hive_metastore/locations.py

```diff
@@ -305,7 +307,14 @@ def _append_records(self, items: Sequence[Table]):
     def _try_load(self) -> Iterable[Table]:
         """Tries to load table information from the database or throws TABLE_OR_VIEW_NOT_FOUND error"""
         for row in self._fetch(
-            f"SELECT * FROM {escape_sql_identifier(self.full_name)} WHERE NOT STARTSWITH(database, 'mounted_')"
+            f"SELECT * FROM {escape_sql_identifier(self.full_name)} WHERE NOT STARTSWITH(database, '{self.TABLE_IN_MOUNT_DB}')"
```

Comment:
Could you elaborate why `_try_load` select tables that are not in `mounted_*` databases?

Reply:
Because this job aims to be incremental. Each time we run this job, we are fetching everything except all tables in mounts from previous run

Reply:
I see, your override of `_append_records` is overwriting the `tables` inventory table. Please make sure to document to let user know this workflow will overwrite inventory table, so don't run it and the assessment workflow at the same time.